app/modules/ads/slots.js

```diff
@@ -379,6 +379,26 @@ export const slots = {
       context.set('slots.incontent_boxad_1.targeting.xna', '0');
     }
   },
+
+  setupCloseButtonTimer() {
+    const { context } = window.Wikia.adEngine;
+
+    context.push('listeners.slot', {
+      onRenderEnded: (adSlot) => {
+        if (['invisible_high_impact_2', 'floor_adhesion'].includes(adSlot.getSlotName())) {
+          const closeButton = document.querySelector('.out-of-page-template.floor-adhesion .button-close');
+
+          if (closeButton) {
+            closeButton.style.display = 'none';
+
+            setTimeout(() => {
```

Comment:
If `custom.timeToCloseButton.invisible_high_impact_2` or `custom.timeToCloseButton. floor_adhesion` are set to `0` it may not be `0s` (because of the event loop) and they'll be displayed for some `ms` ;) But maybe it's good enough for the experiment 🤷‍♂️

Reply:
You right. But anyway this won't be noticeable for the user I guess :)